### DISCUSSION TITLE
Unable to work with tsquery even while using rawQuery

### DISCUSSION THREAD

**Discussion by brookmg** [prisma/prisma#8779] [Q&A] [answered]
I was looking into a full-text search using Prisma but it looks like it's not supported out of the box yet #1684. So I tried to use a raw query to see if it even works 
```js
    const smallQ = await prisma.$queryRaw(`SELECT phraseto_tsquery($1)`, query)
    console.dir(smallQ)
``` 

Unfortunately, it keeps giving me an error 
```
{ query: 'question', limit: 10, before: undefined, page: 0 }
prisma:info Starting a postgresql pool with 9 connections.
prisma:info Started http server on http://127.0.0.1:61284
prisma:query SELECT phraseto_tsquery($1)
PrismaClientKnownRequestError3 [PrismaClientKnownRequestError]:
Invalid `prisma.queryRaw()` invocation:


  Raw query failed. Code: `N/A`. Message: `error deserializing column 0: cannot convert between the Rust type `core::option::Option<alloc::string::String>` and the Postgres type `tsquery``
    at RequestHandler.request (C:\Users\hp\Documents\Github\------------------------------------------\node_modules\@prisma\client\runtime\index.js:36744:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async searchQuestion (C:\Users\hp\Documents\Github\------------------------------------------\dist\controller\content.js:420:18)
    at async C:\Users\hp\Documents\Github\------------------------------------------\dist\routes\v1\question.js:186:18 {
  code: 'P2010',
  clientVersion: '2.29.1',
  meta: {
    code: 'N/A',
    message: 'error deserializing column 0: cannot convert between the Rust type `core::option::Option<alloc::string::String>` and the Postgres type `tsquery`'
  }
}
```

How can one make this work?

**Comment by brookmg** [comment, marked as answer]
Ok, I have made this one work using a workaround mentioned https://github.com/prisma/prisma/issues/5027#issuecomment-759123952

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
